redis/commands/core.py

```diff
@@ -314,6 +314,18 @@ class ManagementCommands:
     Redis management commands
     """
 
+    def auth(self):
+        """
+        The AUTH command authenticates the current connection in two cases:
```

Comment:
I think there's a miscommunication here. The docs should indicate clearly that we're raising this error, since this is intentionally not being implemented.